src/pykiso/cli.py

```diff
@@ -173,6 +173,11 @@ def get_logging_options() -> LogOptions:
     default=None,
     help="allow the user to execute a subset of tests based on branch levels",
 )
+@click.option(
```

Comment:
The CLI help in the readme and getting started should be updated with the new flag